client/src/components/BrandTag/components/SlackBrandTag/SlackBrandTag.tsx

```diff
@@ -0,0 +1,17 @@
+import BrandTag from 'components/BrandTag/BrandTag';
+import slackIcon from 'assets/slack-icon.png';
+
+interface SlackBrandTagProps {
+  name: string;
+}
+
+const SlackBrandTag = ({ name }: SlackBrandTagProps) => {
```

Comment:
I completely do not like repeating code like this where there is no need for it but again nothing important

Reply:
Yeah, after a second thought I agree